rails/app/controllers/dashboard/communities_controller.rb

```diff
@@ -8,6 +8,14 @@ def update
       @community = authorize current_user.community
 
       @community.update(community_params)
+
+      # Append new sponsor logos
+      if file_uploads.present?
+        file_uploads[:sponsor_logos].each do |logo|
+          @community.sponsor_logos.attach(io: logo, filename: logo.original_filename)
+        end
+      end
```

Comment:
Just curious to know more about the `sponsor_logo` related changes. Did the previous sponsor logo behavior break with the upgrade to Ruby 3 / Rails 7?

Reply:
Yeah. Previously, we had `config.active_storage.replace_on_assign_to_many` to false. It allowed us to always append uploaded sponsor logos on assigns (aka: `community.update(params)`). The default behavior is intended to be true, and the flag is being deprecated in 7.1.

The recommendation for appending images on `has_many_attached` is to use `.attach`. I left the GREEN/RED spec for testing that behavior in the code if you're curious to see what the difference is with the settings.